team_b/yappy/lib/services/file_handler.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:io';
+import 'package:flutter/services.dart' show rootBundle;
```

Comment:
Add `import 'package:flutter/foundation.dart';` so you can surround all print statements in `if (kDebugMode)`